docs/modules/operation/pages/quick-start/notifications.adoc

```diff
@@ -45,15 +45,33 @@ It also specifies escalation targets.
 Configuring destination paths separately from individual events lets you use the same information for multiple notifications.
 This minimizes duplication and encourages reuse.
 
-A destination path is required to receive notifications from {page-component-title}.
-The `Email-Admin` destination path is defined by default.
-It sends notifications to the `admin` user, as defined in their notification information (see xref:operation:quick-start/users.adoc#ga-users-next[Users]).
+You must specify a destination path to receive notifications from {page-component-title}.
+The `Email-Admin` destination path is defined by default, and is the notification path when no other destination paths are specified.
+It sends email notifications to users in the Admin group (see xref:operation:quick-start/users.adoc#ga-users-next[Users]).
 
 Follow these steps to define a new destination path:
 
 . Log in as a user with admin privileges.
 . Click the *Gear* icon in the top-right of the screen, and choose menu:Event Management[Configure Notifications > Configure Destination Paths].
-. Click *New Path*, fill in the desired information, and click *Finish*.
+. Click *New Path* and configure it:
+.. Specify a name for the path (for example, Email-Bob) and define an initial delay.
+.. Click *Edit*, and select the users, groups, roles, or email addresses to send notifications to.
+... To add a single email address to the Send to Email Addresses list, click *Add Address*, type the email address, and click *OK*.
+... To remove email addresses from the Send to Email Addresses list, select them from the list and click *Remove Selected Addresses*.
+.. Click *Next Step*.
+.. (Optional) Define a delay interval.
```

Comment:
```suggestion
.. (Optional) Define a delay interval for group notification.
```
Added this information, because I don't get this option when I click "Next Step". It appears only for a group.